genienlp/data_utils/database.py

```diff
@@ -38,25 +38,24 @@
 
 
 class Database(object):
```

Comment:
This class is unused right? It's pre-bootleg and we can kill it. Even your NER experiments will be Bootleg-only.

Reply:
I will kill `RemoteDatabase` but need this one as a baseline against bootleg.